app/modules/usb/scripts/usb.py

```diff
@@ -40,6 +40,8 @@ def flatten_usb_info(array):
                 device['name'] = obj[item]
             elif item == 'vendor_id' or item == 'b_vendor_id':
                 device['vendor_id'] = obj[item]
+            elif item == 'product_id':
```

Comment:
You'll have to make this line be `elif item == 'product_id' or item == 'a_product_id':` to prevent this module from breaking on older systems.